src/transformStateWithClones.js

```diff
@@ -7,7 +7,28 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let newState = { ...state };
+  const arr = [];
+
+  for (const action of actions) {
```

Comment:
Destructure `action` 
```suggestion
  for (const { type, extraData, .... } of actions) {
```